compiler/lib/src/main/scala/codegen/CppWriter/EnumCppWriter.scala

```diff
@@ -189,43 +189,46 @@ case class EnumCppWriter(
       CppDoc.Class.Member.Lines(
         CppDoc.Lines(
           CppDocWriter.writeBannerComment("Constructors"),
-          CppDoc.Lines.Both
         )
       ),
-      CppDoc.Class.Member.Constructor(
-        CppDoc.Class.Constructor(
-          Some(s"Constructor (default value of $defaultValue)"),
-          Nil,
-          Nil,
-          lines(s"this->e = $defaultValue;")
+      CppDoc.Class.Member.Lines(
+        CppDoc.Lines(
+          lines(
+            s"""|
+                |//! Constructor (default value of $defaultValue)
+                |$name()
+                |{
+                |  this->e = $defaultValue;
+                |}"""
+          )
         )
       ),
-      CppDoc.Class.Member.Constructor(
-        CppDoc.Class.Constructor(
-          Some(s"Constructor (user-provided value)"),
-          List(
-            CppDoc.Function.Param(
-              CppDoc.Type("const T"),
-              "e",
-              Some("The raw enum value")
-            )
-          ),
-          Nil,
-          lines("this->e = e;")
+      CppDoc.Class.Member.Lines(
```

Comment:
Can we merge the adjacent Lines members into a single member? I think this would make the code cleaner.

Reply:
Ok, fixed! Both F Prime and FPP functional unit tests still pass.